src/libs/actions/Report.ts

```diff
@@ -1235,14 +1235,12 @@ function deleteReportComment(reportID: string, reportAction: ReportAction) {
         {
             getConflictingRequests: (persistedRequests) => {
                 const conflictingCommands = (
-                    isDeletedParentAction
-                        ? [WRITE_COMMANDS.UPDATE_COMMENT]
-                        : [WRITE_COMMANDS.ADD_COMMENT, WRITE_COMMANDS.ADD_ATTACHMENT, WRITE_COMMANDS.UPDATE_COMMENT, WRITE_COMMANDS.DELETE_COMMENT]
+                    isDeletedParentAction ? [WRITE_COMMANDS.UPDATE_COMMENT] : [WRITE_COMMANDS.ADD_COMMENT, WRITE_COMMANDS.ADD_ATTACHMENT, WRITE_COMMANDS.UPDATE_COMMENT]
                 ) as string[];
                 return persistedRequests.filter((request) => conflictingCommands.includes(request.command) && request.data?.reportActionID === reportActionID);
             },
             handleConflictingRequest: () => Onyx.update(successData),
-            shouldIncludeCurrentRequest: !isDeletedParentAction,
+            shouldIncludeCurrentRequestOnConflict: !isDeletedParentAction,
```

Comment:
Same with above, we need to remove the negation here.
```suggestion
            shouldIncludeCurrentRequestOnConflict: isDeletedParentAction,
```